.gitlab-ci.yml

```diff
@@ -6,7 +6,7 @@ stages:
 variables:
   PREVIEW_CONFIG: "config/preview.yaml"
   LIVE_CONFIG: "config/live.yaml"
-  ARTIFACT_RESOURCE: "public"
+  ARTIFACT_RESOURCE: "dist"
```

Comment:
`corp-ci` references the `public` folder, maybe better to leave all things as `public` in docs so there's less things changing?

Reply:
I accounted for that in a branch on corp-ci, wanted to aim for consistency across docs and corp. https://github.com/DataDog/corp-ci/pull/44

Reply:
I was trying to aim for consistency between corp and docs, I've already made a branch for needed changes to corp-ci. I can make the output folder `public` but I'll also have to change webpack configs. Maybe we can change it to `dist` after the merge then?